notify.go

```diff
@@ -435,6 +519,115 @@ func parseChainNtfnParams(params []json.RawMessage) (*chainhash.Hash,
 	return blockHash, blockHeight, blockTime, nil
 }
 
+// parseFilteredBlockConnectedParams parses out the parameters included in a
+// filteredblockconnected notification.
+//
+// NOTE: This is a btcd extension ported from github.com/decred/dcrrpcclient
+// and requires a websocket connection.
+func parseFilteredBlockConnectedParams(params []json.RawMessage) (int32,
+	*wire.BlockHeader, []*btcutil.Tx, error) {
+
+	if len(params) != 3 {
```

Comment:
ACK. I left it as `!=` to keep it the same as the current standard, but since the plan is to make all the other handlers use `<`, it makes sense to go ahead and do it right in the new code.